guake/terminal.py

```diff
@@ -121,6 +121,10 @@ def __init__(self, guake):
 
         self.setup_drag_and_drop()
 
+        exclude = {"GDK_BACKEND"}
```

Comment:
we may put this as a constant?

```
self.ENVV_EXCLUDE_LIST = ["GDK_BACKEND"]
```

Reply:
Alright, sure, could be useful at some point.